### PR TITLE
Add devtips

### PR DESCRIPTION
**Pull request by xtrp**
- **Name of the youtuber:** DevTips
- **What the channel is about (eg. web development, design, ...)**: Full-Stack Web Development with a focus on designing with CSS
- **Which section is the channel in? (if you created a section, please specify why)**: Web Development
- **Why do you consider the youtuber deserves a place in this list? *What does make it awesome?***: The channel includes many videos on a number of web development topics and many popular tutorials on CSS, styling, and animations in the browser.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -256,6 +263,13 @@ Content about: CSS \
 Featured playlists: `Full Course: Level Up Your CSS Animation Skills`. \
 <br/>
 
+[<img align="left" height="94px" width="94px" alt="DevTips channel's avatar" src="https://yt3.ggpht.com/a/AATXAJxNH7UVf0iZB1vbfP-n_r4vnuWpVlzRY5eKoJqeyw"/>](https://www.youtube.com/c/DevTipsForDesigners/)
+
+[**DevTips**](https://www.youtube.com/c/DevTipsForDesigners/) [<img height="16px" width="16px" alt="Badge for verified YouTube channels" src="media/badge-verified.svg" title="Is a verified YouTube channel"/>](badges.md#verified-youtube-channel) \
```
Comment (JoseDeFreitas) [inline readme.md:268]: "DevTips" is not a verified YouTube channel.

File: readme.md [hunk 2]
```diff
@@ -104,6 +104,13 @@ Featured playlists: `Building an 8-bit breadboard computer!`, `Network tutorial`
 Content about: Algorithms, Various programming languages \
 Featured playlists: `The Nature of Code: Simulating Natural Systems with Processing`, `Learning Processing: A Beginner's Guide to Programming Images, Animation, and Interaction`.
 
+[<img align="left" height="94px" width="94px" alt="Mike Dane channel's avatar" src="https://yt3.ggpht.com/a/AATXAJwZDjZBLse-ByxN1EAxA5ll42m2QN_8UEL_J0wbdA"/>](https://www.youtube.com/c/GiraffeAcademy/)
+
+[**Mike Dane**](https://www.youtube.com/c/GiraffeAcademy/) [<img height="16px" width="16px" alt="Badge for verified YouTube channels" src="media/badge-verified.svg" title="Is a verified YouTube channel"/>](badges.md#verified-youtube-channel)
```
Comment (JoseDeFreitas) [inline readme.md:109]: "Mike Dane" is not a verified YouTube channel.

**Review by JoseDeFreitas — CHANGES_REQUESTED** [review]
I've added two comments about the badges. As they don't fit the criteria, they should have no badge.

Apart from that, as stated in the [guidelines](https://github.com/JoseDeFreitas/awesome-youtubers/blob/main/contributing.md), the channels should be added at the bottom of the section they are. This is to follow the way it is in the [awesome](https://github.com/sindresorhus/awesome) project and also to not have discussions about "why is this YouTuber above this one" or something like that.

Writing about the YouTubers you've added, they are good! I've watched some videos from both of them. Thank you!

**Comment by xtrp** [comment]
I've just made the changes you requested on both PRs. Please let me know if there are any other problems. Thanks.

— Gabriel Romualdo

**Comment by JoseDeFreitas** [comment]
Hello @xtrp! The issues are fixed now. Thank you for the PRs and thank you for adding those awesome YouTube channels 😄.

**Pull request merged**